SU-SWS/decanter
# D8CORE-2536: adding aria-label

# READY for review.

# Summary
- Adds `<span class="su-global-footer__link--external">(link is external)</span>` to the external links in the global footer

# Needed By (Date)
Soon

# Urgency
Not urgent

# Steps to Test

1. Check out this branch in Stanford Basic
1. run `npm install` in Stanford Basic
1. Run `npm run publish` to populate /decanter in the Stanford Basic templates directory
1. Inspect global footer to verify `<span class="su-global-footer__link--external">(link is external)</span>` accompanies every external link in the global footer.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?
D8Core, Decanter, anything using Decanter

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2536
- Related PRs: https://github.com/SU-SWS/decanter/pull/717
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
